integration/testcmd/mountkexec/uinit/mountkexec.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2018 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"log"
+	"os/exec"
+
+	"github.com/u-root/webboot/pkg/mountkexec"
+	"golang.org/x/sys/unix"
+)
+
+// Mount a vfat volume and kexec the kernel within.
+func main() {
+	// Test : Mount on non-existent directoty
+	if err := mountkexec.MountISO("/dev/sda1", ""); err != nil {
+		log.Printf(err.Error())
+	}
+	// Test : Mount non-existent iso
+	if err := mountkexec.MountISO("non-existent file", "mountDir"); err != nil {
+		log.Printf(err.Error())
+	}
+	// Test : Mount on existing dir and an existing iso
+	if err := mountkexec.MountISO("/dev/sda1", "mountDir"); err != nil {
+		log.Fatal(err)
```

Comment:
this could also be a table, instead of blocks of repeated code.

Reply:
Done.